lib/naughty/core.lua

```diff
@@ -603,6 +613,10 @@ function naughty.notify(args)
                                bg = bg,
                                border_color = border_color,
                                border_width = border_width,
+                               shape_border_color = shape and border_color,
+                               shape_border_width = shape and border_width,
+                               shape = shape,
+                               shape_args = shape_args,
```

Comment:
btw, shape_args seems to be ignored now by wibox, so in case if it's not going to be changed, in naughty it should be changed like:

```lua
shape = function(c, w, h) return shape(c, w, h, table.unpack(shape_args)) end
```